### ISSUE TITLE
Merge paraurethral gland and skene gland

### ISSUE THREAD

**Issue by cmungall** [obophenotype/uberon#1292]
From #1243 

> minor vestibular gland UBERON:0000461
> 
> Incorrect superclass: paraurethral gland. This is actually a synonym and not a broader term.

We have these notes:

```
property_value: external_ontology_notes "in MP female urethral gland, Skene gland and paraurethral gland are exact syns. Note that NCITA seems to use the term in a more generic way, as a parent for skene gland, cowper gland (which may be the same as the male bulbo-urethral gland)" xsd:string {external_ontology="MP"}
```

Here is NCIT:

```
   is_a NCIT:C12339 ! paraurethral gland *** 
      is_a NCIT:C32395 ! cowper gland
      is_a NCIT:C49311 ! gland of littre
      is_a NCIT:C61122 ! skene gland
```

I think NCIT is incorrect in its terminology; 'paraurethral' refers to a particular female gland, NCIT is using it generically. Trying to align with it created a situation whereby we artificially separated paraurethral and skene.

Also of relevance: https://github.com/obophenotype/mammalian-phenotype-ontology/issues/1272

**Comment by NCIEVS** [comment]
@cmungall , I've just alerted the NCIt editing group.  Thanks!

**Comment by cmungall** [comment]
FMA has distinct classes also:

```
  is_a FMA:61775 ! physical anatomical entity
   is_a FMA:67165 ! material anatomical entity
    is_a FMA:305751 ! anatomical structure
     is_a FMA:67135 ! postnatal anatomical structure
      is_a FMA:67498 ! organ
       is_a FMA:55670 ! solid organ
        is_a FMA:55661 ! parenchymatous organ
         is_a FMA:55662 ! lobular organ
          is_a FMA:20020 ! vestibular gland
           is_a FMA:20019 ! lesser vestibular gland ***  [part_of: "urogenital part of female perineum"] [part_of: "female external genitalia"]
      is_a FMA:82472 ! cardinal organ part
       is_a FMA:14065 ! organ component
        is_a FMA:7146 ! organ component gland
         is_a FMA:20083 ! paraurethral gland ***  [part_of: "urogenital part of female perineum"]
          is_a FMA:20084 ! right paraurethral gland
          is_a FMA:20085 ! left paraurethral gland

```

**Comment by cmungall** [comment]
fixed in #1282
